src/jit/block.cpp

```diff
@@ -637,3 +637,69 @@ unsigned PtrKeyFuncs<BasicBlock>::GetHashCode(const BasicBlock* ptr)
     return ptr->bbNum;
 }
 
+//------------------------------------------------------------------------
+// inlExpLst: default constructor for an inlExpList
+//
+// Notes: use for the root instance. We set ixlCode to nullptr here
+// (rather than the IL buffer address of the root method) to preserve
+// existing behavior, which is to allow one recursive inline.
+
+inlExpLst::inlExpLst() : ixlParent(nullptr), ixlChild(nullptr), 
+                         ixlSibling(nullptr), ilOffset(BAD_IL_OFFSET), 
+                         ixlCode(nullptr)
+{
```

Comment:
Sorry about that. I'll be back into this code soon enough so will hold off on fixing just this bit.

Coding conventions without some kind of tooling support are a problem for those of us that work in many different code bases.

Clang-format is probably configurable enough to support the jit's formatting style. We should really look into this.

There are also existing violations of the CC's which could perhaps be automatically fixed.